### PR TITLE
[zh] Fix 2 missing feature state

### PR DESCRIPTION
**Pull request by Sea-n**
There are 2 missing feature states in @Arhell's previous PR.

## Related PR
- #33177

### CODE REVIEW

**Comment by Sea-n** [comment]
Note:
According to diff result, some sentences in the comment is not exactly the same with English version. This might need to be fix with another issue.

**Comment by tengqm** [comment]
@Sea-n With the 'main' branch, we are still talking about 1.23 version, not 1.24. Looks like you are introducing something specific to 1.24 in this PR, into 'main'?

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -177,6 +177,7 @@ different Kubernetes components.
 | `GracefulNodeShutdown` | `false` | Alpha | 1.20 | 1.20 |
 | `GracefulNodeShutdown` | `true` | Beta | 1.21 | |
 | `GracefulNodeShutdownBasedOnPodPriority` | `false` | Alpha | 1.23 | |
+| `GracefulNodeShutdownBasedOnPodPriority` | `true` | Beta | 1.24 | |
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:180]: This looks like a mistake in the English upstream.

**Comment by tengqm** [comment]
Please limit this PR to the fix of `ServerSideFieldValidation` feature.
The `ControllerManagerLeaderMigration` one should be fixed in the English upstream.

**Pull request merged**